tests/lite/test_wrappers.py

```diff
@@ -145,8 +146,10 @@ def test_lite_optimizer_wraps():
 def test_lite_optimizer_steps():
     """Test that the LiteOptimizer forwards the step() and zero_grad() calls to the wrapped optimizer."""
     optimizer = Mock()
-    accelerator = Mock()
+    strategy = Mock()
+    accelerator = Accelerator(strategy)
```

Comment:
this is still outdated from a previous verson of your code where the first argument was the strategy, but now it is the precision plugin

```suggestion
    accelerator = Accelerator(training_type_plugin=strategy)
```